.travis.yml

```diff
@@ -108,6 +108,14 @@ before_script:
       conda install swig; cd ../;
       git clone https://github.com/opesci/yask.git;
       cd yask; make compiler-api; pip install -e .; cd ../devito;
+    elif [[ $DEVITO_BACKEND == 'ops' ]]; then
+      pwd;
+      git clone https://github.com/opesci/OPS.git;
+      cd OPS/ops;
+      export OPS_INSTALL_PATH=$(pwd);
+      cd c;
+      make;
```

Comment:
CC=gcc-7 make and should be gtg

Reply:
thank you, let me try that.
actually, the `make` seems to be working fine.. apparently the problem is that i'm not returning to the correct devito's installation path after "making"

i tried: `../../` , `$HOME` , `/` , etc...
now i'm trying `$HOME/opesci/devito`

Reply:
`$HOME/opesci/devito` well that's not gonna work

Reply:
every trial takes me like 1,5h >.< 

glad you brought that..

let me try with:
```
      cd ../;
      git clone https://github.com/opesci/OPS.git;
      cd OPS/ops/c;
      CC=gcc-7 make;
      cd ../../../devito;
```